deploy/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+    name: openshift-cluster-ingress-operator
+    labels:
+        name: openshift-cluster-ingress-operator
```

Comment:
Why is the label needed? I don't see similar labels on other operators' namespaces.

Reply:
I added it just for information purposes, i'll remove it so we have the basic namespace configuration.